pkg/dev-env/docker-compose.yml

```diff
@@ -157,15 +241,15 @@ services:
       POSTGRES_DB: postgres
       POSTGRES_PASSWORD: postgres
 
-  ## Gamedao graph service
+  ## GameDAO graph service
   gamedao-graph-service:
     build: '../service'
     environment:
       IPFS_API_URL: "https://ipfs.infura.io:5001/api/v0"
       IPFS_CLIENT_ID: "1v04L2wj5JmI0JgKF5KztV0oN8o"
```

Comment:
you should not expose access credentials in a public repository. these belong into a .env not checked in or into a proper key management...

Reply:
Oh... I've totally overseen that... Mb, I have just updated it, not sure why there are credentials in the compose file.
Will fix it.